server/controllers/middlewares/index.js

```diff
@@ -1,4 +1,4 @@
 const { auth } = require('./auth');
+const { teacherProfile } = require('../routes/teacherProfile');
```

Comment:
don't think this should be here, it's not a middleware :) it's (correctly) exported through `controllers/index.js`